.github/workflows/ci.yml

```diff
@@ -10,12 +10,12 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
 
       - name: Setup Node.js
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v2
         with:
-          node-version: 12.x
+          node-version: 16
```

Comment:
Nitpick but will be good to pin specific complete versions. This install the latest in the series and any future release can break CI here. Not a big deal now since the latest series is 17.x